modules/core/site.js

```diff
@@ -2498,3 +2511,29 @@ const observeMessageTextMutationAndHandleExternalResources = (inline) => {
         });
     }
 };
+
+var setup_nexter_date = function(callback) {
```

Comment:
For the naming convention, it’s better to stick to camel case for JavaScript code and avoid using `var` when creating functions and variables. Instead, declare them as named functions, as functions are hoisted and available in any JavaScript file where they are used.